app/plugins/adp/src/components/DeliveryProgrammeAdmin/DeliveryProgrammeAdminViewPage.tsx

```diff
@@ -13,6 +13,8 @@ import {
   useErrorCallback,
 } from '../../hooks';
 import { DefaultTable } from '../../utils';
+import { AddProgrammeAdminButton } from './AddProgrammeAdminButton';
+import AddBoxIcon from '@mui/icons-material/AddBox';
```

Comment:
I was looking into this in my PR, seems that @mui/* is for v5, while @material-ui/* is for v4. Backstage supports both, but I think we should stick to one or the other. v4 seems to be most common in other files, so ive reverted some places back to v4 in my PR.
```suggestion
import AddBoxIcon from '@material-ui/icons/AddBox';
```